app/src/main/java/de/tu_darmstadt/seemoo/nfcgate/nfc/config/ConfigBuilder.java

```diff
@@ -28,6 +28,8 @@ public void add(ConfigOption option) {
         mOptions.add(option);
     }
 
+    public void addAll(ConfigBuilder builder) { mOptions.addAll(builder.getOptions()); }
```

Comment:
Please format this method in line with the rest of the file
```suggestion
    public void addAll(ConfigBuilder builder) {
        mOptions.addAll(builder.getOptions()); 
    }
```